src/Testing/src/xunit/HelixConstants.cs

```diff
@@ -6,6 +6,6 @@ namespace Microsoft.AspNetCore.Testing;
 public static class HelixConstants
 {
     public const string Windows10Arm64 = "Windows.10.Arm64v8.Open;";
-    public const string DebianArm64 = "Debian.9.Arm64.Open;";
+    public const string DebianArm64 = "Debian.11.Arm64.Open;";
```

Comment:
We skip some tests on Debian 9 Arm64 e.g. see #28090. May be worth testing if they work on Debian 11 Arm64.

Reply:
@Tratcher any chance #28090 can be avoided on Debian 11 Arm64❔

Reply:
/ping @Tratcher

Reply:
I'll mention this in the issue in case we want to un-skip those in a separate PR

Reply:
Ah, you overlapped me pinging @Tratcher again 😃

Reply:
I don't have an answer, I'm not sure what the issue is.

Reply:
:ok: thanks. I'll leave the skips on Helix for #28090 alone in this PR.